dpnp/dparray.pyx

```diff
@@ -281,6 +372,26 @@ cdef class dparray:
 
         return interface_dict
 
+
+    @property
+    def __sycl_usm_array_interface__(self):
```

Comment:
This is a code duplication to `__array_interface__`. I see one field diffrence only. It looks like it is better to add this field into `__array_interface__`

Reply:
No, this is implementation of a different mechanism, which should not be confused with `__array_interface__`. 

It also serves as a signal that an object `pyobj` can export USM memory, detectable via `hasattr(pyobj, "__sycl_usm_array_interface__")`.